testarch/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ define_arch(
   --write_rr_graph \${OUT_RRXML_REAL}"
   NO_PINS
   NO_BITSTREAM
+  ROUTE_CHAN_WIDTH 100
```

Comment:
Why is this needed? Everything needs a `ROUTE_CHAN_WIDTH` now?

Reply:
Because the required ROUTE_CHAN_WIDTH varies by arch.